cluster-provision/k8s/1.31/manifests/prometheus/prometheus/prometheus-prometheus.yaml

```diff
@@ -16,34 +16,35 @@ spec:
       name: alertmanager-main
       namespace: monitoring
       port: web
+  enableFeatures: []
   externalLabels: {}
-  image: quay.io/prometheus/prometheus:v2.26.0
+  image: quay.io/prometheus/prometheus:v2.53.0
   nodeSelector:
     kubernetes.io/os: linux
   podMetadata:
     labels:
       app.kubernetes.io/component: prometheus
+      app.kubernetes.io/instance: k8s
       app.kubernetes.io/name: prometheus
       app.kubernetes.io/part-of: kube-prometheus
-      app.kubernetes.io/version: 2.26.0
+      app.kubernetes.io/version: 2.53.0
   podMonitorNamespaceSelector: {}
   podMonitorSelector: {}
   probeNamespaceSelector: {}
   probeSelector: {}
-  replicas: 1
+  replicas: 2
```

Comment:
I think we just need one replica here as this is just a test cluster

Reply:
@brianmcarey done